src/snowflake/snowpark/dataframe.py

```diff
@@ -3772,6 +3772,49 @@ def convert(col: ColumnOrName) -> Expression:
         exprs = [convert(col) for col in parse_positional_args_to_list(*cols)]
         return exprs
 
+    # class property to control the number of rows to diplay
+    __rows = 50
```

Comment:
Is this naming convention followed by ipython variables? I think there should be a better way to set this property because users should not be accessing variables that start with `_`.

Reply:
No, I was just thinking on a way to expose a mechanism for setting the number of rows to display by default. Maybe a more explicit value in the session object for example will be better.